src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -276,13 +276,19 @@ export class PiaIntakeService {
     }
 
     // Scenario 3: As a CPO, retrieve all pia-intakes in CPO_Review
+    // CPO can only see CPO_REVIEW status in the Active PIAs
+    // And COMPLETED PIAs in the completed tab - when filterByStatus === COMPLETE
     if (isCPO) {
       if (
         query.filterByStatus &&
-        query.filterByStatus !== PiaIntakeStatusEnum.CPO_REVIEW
+        ![
```

Comment:
instead of NOT, would it be more readable to include all the status it needs to check?